packages/@react-spectrum/s2/src/SegmentedControl.tsx

```diff
@@ -125,11 +140,11 @@ function SegmentedControl(props: SegmentedControlProps, ref: DOMRef<HTMLDivEleme
   )
 }
 
-function DefaultSelectionTracker(props) {
+function DefaultSelectionTracker(props: DefaultSelectionTrackProps) {
   let state = useContext(RadioGroupStateContext);
   let isRegistered = useRef(!(props.defaultValue == null && props.value == null));
 
-  let register = useCallback((value) => {
+  let register = useCallback((value: string) => {
```

Comment:
string or number i believe

Reply:
in our docs it looks like defaultValue and value are defined as `string | null`

Reply:
Ah, right, hasn't gone in yet [fix: add number type in value](https://github.com/adobe/react-spectrum/pull/6636#top)